### PR TITLE
docs: Describe annual renewal process for CC commitment

### PR DESCRIPTION
**Pull request by sarina**
Moving forward, the Axim team will engage in annual audits of individual Core Contributors as well as organizational commitments to the Core Contributor Program. These edits describe when these audits will take place. 

Please provide feedback by 18 September.

### CODE REVIEW

**Review by feanil — APPROVED** [review]

File: oeps/processes/oep-0054-core-contributors.rst [hunk 1]
```diff
@@ -118,6 +118,26 @@ Questions about the Program can be directed to the Axim administrator(s) via
...
+
+Organizations will be asked to affirm and/or revise their commitment in the
+`Declaration of Commitment to the Core Contributor Program`_ each year in December,
+for the following calendar year (January through December).
+
+Individuals will be asked to affirm their continued commitment to and
+involvement in the program each year in January, for the following calendar year
+(February through January). Individuals who are unable to affirm their
+commitment will be removed from the program as described in `removing ccs`_, and
+are more than welcome to re-join the Program if and when their time allows them
```
[the remark this answers is not in the document]
Comment (sarina) [reply oeps/processes/oep-0054-core-contributors.rst:136]: It probably does, I'm not certain I feel making that change belongs in this particular changeset though.

File: oeps/processes/oep-0054-core-contributors.rst [hunk 2]
```diff
@@ -118,6 +118,26 @@ Questions about the Program can be directed to the Axim administrator(s) via
 the ``#core-contributors`` Slack room in the `Open edX Slack
 <https://openedx.slack.com/>`_.
 
+Renewing Organizational and Personal Commitments
+------------------------------------------------
+
+Each year, the Program Administrators will conduct a yearly commitment renewal
```
[the remark this answers is not in the document]
Comment (sarina) [reply oeps/processes/oep-0054-core-contributors.rst:124]: I think I was trying to say both that the audit was conducted each year, _and_ that the commitment each org needs to make should be a year-long commitment. Does that come through if I drop the "Each year," ?

[the remark this answers is not in the document]
Comment (bmtcril) [reply oeps/processes/oep-0054-core-contributors.rst:124]: Sure, that works 👍

**Comment by antoviaque** [comment]
@sarina Reposting my comment from the forum here, as requested :)

Thank you for the update! :+1: Annual reviews make sense - the original goal of once per quarter was a bit ambitious, it’s a lot of work and discussions to do these reviews. Switching to annual reviews might help to do them more reliably.

Reviews are important, because it’s important for the project that all partner organizations contribute a share, in proportion to our respective sizes - we all benefit from the combined effort when everyone reliably chips in. Making sure that we regularly check & renew organizational commitments helps maintaining a steady flow of contributed work, which the project relies on.

I know of many core contributors, now or in the past, that would have wanted to contribute more, but didn’t get enough time dedicated to it by their organization. It might help to bring up the topic at the organization level.

**Pull request merged**